pkg/webui/console/containers/notifications-dashboard-panel/index.js

```diff
@@ -105,15 +113,25 @@ const NotificationsDashboardPanel = () => {
       messageDecorators={totalUnseenNotifications > 0 ? <MessageDecorator /> : undefined}
       className={style.notificationPanel}
     >
-      <FetchTable
-        entity="notifications"
-        headers={headers}
-        pageSize={5}
-        baseDataSelector={baseDataSelectors}
-        getItemsAction={getItems}
-        getItemPathPrefix={item => `/notifications/inbox/${item.id}`}
-        paginated={false}
-      />
+      {notifications && notifications.length === 0 ? (
+        <div className="p-vert-ls-l flex-grow j-center">
```

Comment:
```suggestion
        <div className="flex-grow j-center">
```
Let's not work with a padding but a minimum height on the overall panel style rather.